docs/contributing.rst

```diff
@@ -12,10 +12,30 @@ This is a `Jazzband <https://jazzband.co>`_ project. By contributing you agree t
 Setup
 =====
 
-Fork ``django-oauth-toolkit`` repository on `GitHub <https://github.com/jazzband/django-oauth-toolkit>`_ and follow these steps:
 
- * Create a virtualenv and activate it
- * Clone your repository locally
+Fork ``django-oauth-toolkit`` repository on `GitHub
+<https://github.com/jazzband/django-oauth-toolkit>`_ and clone to your
+development machine. The recommended method to manage your development
```

Comment:
`uv` looks interesting but this is separate from the bug you're trying to address. I'd say open an issue for discussion of uv if you think it is something the project could benefit from and why you think it is better than other project tools like pip, poety, pipenv, hatch, pdm, rye, etc.

Reply:
Apologies, I wrongly assumed that uv was the detault tool being used by package management. I **know** that it isn't poetry or pipenv because these tools have their own format requirements, but I should have considered that standard setuptools should be enough.